fe4u-boilerplate-main/src/css/style.css

```diff
@@ -0,0 +1,721 @@
+/* stylelint-disable */
+/*-------------------------NORMALIZE-----------------------*/
+*,
+::after,
+::before {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "Open Sans", sans-serif;
+}
+
+a {
+  text-decoration: none;
+  display: block;
+}
+
+input,
+textarea {
+  border: 1px solid black;
+  border-radius: 3px;
+
+}
+
+select {
+  border: 1px solid black;
+  border-radius: 3px 3px 0 0;
+}
+
+input:active, input:hover, input:focus {
+  outline: 0;
+  outline-offset: 0;
+}
+
+table {
+  border-collapse: collapse;
+}
+
+hr {
+  margin: 15px 2vh;
+}
+
+.canvas-content {
+  display: flex;
+  max-width: 300px;
+  align-content: center;
+}
+
+.canvas-list {
+  display: grid;
+  justify-items: center;
+  grid-template-columns: repeat(auto-fill, minmax(50%, 1fr));
+}
+
+/*-------------------------FONT-SIZE-----------------------*/
+a, p, label, td, .button, input, select {
+  font-size: min(12px, 3vw);
+}
+
+h3, th, summary {
+  font-size: min(18px, 3.5vw);
+}
+
+h2 {
+  font-size: min(22px, 4.5vw);
+}
+
+.user-dont-select,
+h1,
+h2,
+h3,
+summary {
+  user-select: none;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+
+}
+
+/*-------------------------ICON Favicon-----------------------*/
+i {
+  cursor: pointer;
+}
+
+i.li {
+  font-size: 16px;
+}
+
+i.close {
+  padding-left: 15px;
+}
+
+i.ibutton {
+  font-size: 30px;
+}
+
+i.toggle {
+  font-size: 24px;
+}
+
+i.favorite {
+  position: absolute;
+  right: 5px;
+  top: 5px;
+}
+
+#ifavorite {
+  font-size: 20px;
+}
+
+i.close:hover:hover,
+#ifavorite:hover,
+i.toggle:hover {
+  color: #eb1337;
+}
+
+i.selected {
+  color: rgb(36, 180, 144)
+}
+
+
+div.icon-content {
+  display: flex;
+  align-items: center;
+}
+
+a.logo {
+  color: #000;
+}
+
+/*-------------------------Block-menu header-----------------------*/
+div.header {
+  padding-top: 10px;
+}
+
+.header-logo {
+  align-items: center;
+  display: flex;
+  margin-left: 15px;
+  color: black;
+}
+
+/*-------------------------CONTENT Block-menu-----------------------*/
+.menu {
+  text-align: center;
+  margin-bottom: 10px;
+  padding: 0 10px;
+}
+
+.background {
+  filter: blur(3px) contrast(.8);
+  -webkit-filter: blur(3px) contrast(.8);
+  background-color: rgba(0, 0, 0, 0.15);
+  transition: -webkit-filter 0.5s ease-in-out;
+  transition: background-color 0.5s ease-in-out;
+}
+
+div.contents {
+  max-width: 85%;
+  margin-left: auto;
+  margin-right: auto;
+  box-shadow: 0 4px 8px 2px rgba(155, 155, 155, 0.2);
+}
+
+.flex {
```

Comment:
What if you will need the class with the next styles?
```
  display: flex;
  justify-content: flex-start;
  align-items: center;
```

This class name is too general and non informative. If you need to set special styles for `flex`, consider Bootstrap way:
```
.d-flex {
  display: flex;
}

.justify-between {
  justify-content: space-between;
}

.align-center {
  align-items: center;
}
```